### PR TITLE
Add convert_names_case to make_federated_schema

### PR DESCRIPTION
**Pull request by a-rhote**
Does what it says on the tin

### CODE REVIEW

**Review by rafalp — APPROVED** [review]

**Comment by a-rhote** [comment]
@rafalp should I put it under a specific version or heading?

**Comment by rafalp** [comment]
Under next unreleased version

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## Next Unreleased Version
+- Added `convert_names_case` option to `make_federated_schema`.
```
Comment (rafalp) [inline CHANGELOG.md:4]: Please rebase on master first.

Comment (a-rhote) [reply CHANGELOG.md:4]: Can do!

Comment (a-rhote) [reply CHANGELOG.md]: Ugh, Github did something unexpected when I clicked an update button, give me a moment to update

**Comment by a-rhote** [comment]
There we go, should be rebased to master and fixed. Apologies for the headache

**Review by rafalp — APPROVED** [review]

**Comment by rafalp** [comment]
Excellent. Thank you!

**Comment by rafalp** [comment]
Tests suite has exploded. Reverting.

**Comment by a-rhote** [comment]
Oh weird, I pulled that right out of the make_executable_schema def, maybe I just missed the import at the top

**Pull request merged**